qiskit/optimization/problems/quadratic_program.py

```diff
@@ -1093,6 +1093,62 @@ def from_ising(self,
         offset -= offset
 
 
+    def is_feasible(self, x: Union[List[float], np.ndarray], detailed = False) -> bool:
+
+        satisfied_variables = []
+        for i, v in enumerate(x):    
+            variable = self.get_variable(i)
+            if variable._lowerbound <= v <= variable._upperbound :
+                satisfied_variables.append(True)
+                # print(f'{variable._name} is within the bounds') 
+            else :
+                satisfied_variables.append(False)
+                # print(f'{variable._name} is outside the bounds')
+
+        satisfied_linear_constraints = []
+        for constraint in self._linear_constraints:
+            lhs = constraint.evaluate(x)
+            if constraint.sense == ConstraintSense.LE:
+                satisfied_linear_constraints.append(lhs <= constraint.rhs)
+            elif constraint.sense == ConstraintSense.GE:
+                satisfied_linear_constraints.append(lhs >= constraint.rhs)
+            elif constraint.sense == ConstraintSense.EQ:
+                satisfied_linear_constraints.append(isclose(lhs, constraint.rhs))
+            else:
+                raise QiskitOptimizationError("Invalid sense!")
+
+        satisfied_quadratic_constraints = []    
+        for constraint in self._quadratic_constraints:
+            lhs = constraint.evaluate(x)
+            if constraint.sense == ConstraintSense.LE:
+                satisfied_quadratic_constraints.append(lhs <= constraint.rhs)
+            elif constraint.sense == ConstraintSense.GE:
+                satisfied_quadratic_constraints.append(lhs >= constraint.rhs)
+            elif constraint.sense == ConstraintSense.EQ:
+                satisfied_quadratic_constraints.append(isclose(lhs, constraint.rhs))
+            else:
+                raise QiskitOptimizationError("Invalid sense!")
+            
+            
+        if(detailed == True):
+
+            for i in range(len(satisfied_variables)):
+                (print(f'{self.get_variable(i)._name} is within the bounds') if satisfied_variables[i]
+                else logger.warning(f'{self.get_variable(i)._name} is outside the bounds'))
+
+            for i in range(len(satisfied_linear_constraints)):
+                (print(f'{self.get_linear_constraint(i).name} is statisfied') if satisfied_linear_constraints[i] 
+                else logger.warning(f'{self.get_linear_constraint(i).name} is not statisfied'))
+
+            for i in range(len(satisfied_quadratic_constraints)):
+                (print(f'{self.get_quadratic_constraint(i).name} is statisfied') if satisfied_linear_constraints[i] 
+                else logger.warning(f'{self.get_quadratic_constraint(i).name} is not statisfied'))
+
+        return (np.sum(satisfied_variables) == len(self._variables) and
```

Comment:
Better `self._get_num_vars()` instead of `len(self._variables)`, but the latter is implemented as `len` anyway.